designs/cloudWatch/cloudWatchIntegration.md

```diff
@@ -0,0 +1,62 @@
+# CloudWatch Logs Experience
+
+* **Type**: UX Design
+* **Status**: Proposal
+
+## Introduction
+CloudWatch Logs enables a user to centralize the logs from all of their systems, applications, and AWS services that they use, in a single, highly scalable service. This document describes adding CloudWatch Logs into the JetBrains toolkit in order to facilitate easily viewing and searching them for specific error codes or patterns.
+
+## Comparison to other products
+
+| Product                       | Difference    |
+|-------------------------------|---------------|
+| AWS Toolkit for Visual Studio | Not supported |
+| AWS Toolkit for VSCode        | Not supported |
+
+## Experience
+
+### View Log Groups
+
+#### Alternative
+Instead of building a new CloudWatch service node in the AWS Explorer, we instead build a CloudWatch Log Group browser tab in the CloudWatch tool window. This new tab would also be linked to the active AWS connection settings. This would differ from the other tabs which may lead to a confusing user experience and seen as a con.
+
+![ALT_BROWSE_LOG_GROUP]*Possible alternative for browsing Log Groups*
+
+### View Log Streams
+Upon selecting a Log Group, the user would be shown a new tab in the CloudWatch tool window. This new tab will list all the streams for the requested group to allow the user to find the exact stream they are looking for.
```

Comment:
Are we going to add a way to sort them?

Reply:
Oh, ya. I forgot to call it out, but you can see the sort caret in the UI mock. Will call it out